metadata_backend/api/handlers/restapi.py

```diff
@@ -91,8 +91,6 @@ async def _handle_check_ownership(self, req: Request, collection: str, accession
 
             submission_op = SubmissionOperator(db_client)
             check, submission_id, _ = await submission_op.check_object_in_submission(collection, accession_id)
-            # if published:
-            #     _check = True
```

Comment:
I couldn't figure out a reason for having this. Do we need to have it when it was already commented out?

Reply:
i vote to remove